src/os/ObjectStore.h

```diff
@@ -1362,6 +1367,34 @@ class ObjectStore {
       }
       data.ops++;
     }
+
+  /*
+   * Move source object to base object.
+   * Data portion is only copied from source object to base object.
+   * The copy is done according to the move_info vector of tuple, which
+   * has information of src_offset, dest_offset and length.
+   * Finally, the source object is deleted.
+   */
+  void move_ranges_destroy_src(const coll_t& cid, const ghobject_t& src_oid, ghobject_t oid,
+                     const vector<boost::tuple<uint64_t, uint64_t, uint64_t>> move_info) {
+    if (use_tbl) {
+      __u32 op = OP_MERGE_DELETE;
+      ::encode(op, tbl);
+      ::encode(src_oid, tbl);
+      ::encode(cid, tbl);
+      ::encode(oid, tbl);
+      ::encode(move_info, tbl);
+    } else {
```

Comment:
you can drop the use_tbl case here; it's about to be ripped out anyway and will just cause a merge conflict if it's included here.